doc/rados/operations/bluestore-migration.rst

```diff
@@ -2,68 +2,67 @@
  BlueStore Migration
 =====================
 
-Each OSD can run either BlueStore or Filestore, and a single Ceph
-cluster can contain a mix of both.  Users who have previously deployed
-Filestore OSDs should transition to BlueStore in order to
-take advantage of the improved performance and robustness.  Moreover,
-Ceph releases beginning with Reef do not support Filestore. There are
-several strategies for making such a transition.
-
-An individual OSD cannot be converted in place;
-BlueStore and Filestore are simply too different for that to be
-feasible.  The conversion process uses either the cluster's normal
-replication and healing support or tools and strategies that copy OSD
-content from an old (Filestore) device to a new (BlueStore) one.
-
-
-Deploy new OSDs with BlueStore
-==============================
-
-New OSDs (e.g., when the cluster is expanded) should be deployed
-using BlueStore.  This is the default behavior so no specific change
-is needed.
-
-Similarly, any OSDs that are reprovisioned after replacing a failed drive
-should use BlueStore.
-
-Convert existing OSDs
-=====================
-
-Mark out and replace
+Each OSD is capable of running either BlueStore or Filestore, and an
+individual Ceph cluster can operate with a mixture of both.
+Because BlueStore is superior to Filestore in performance and
+robustness, and because Filestore is not supported by Ceph releases
+beginning with Reef, users deploying Filestore OSDs should transition
```

Comment:
I added this recently, but we’d do well to verify the actual status of filestore in reef

Reply:
@neha-ojha: Does Reef support Filestore?

Reply:
Replying to myself: No, Reef does not support Filestore. See this commit: 

https://github.com/ceph/ceph/commit/405f402b5764507ca481e2e81b62b32e5a5ecaaf